Source/WebCore/platform/graphics/DecodingOptions.h

```diff
@@ -69,9 +78,12 @@ class DecodingOptions {
         return sizeForDrawing()->maxDimension() >= other.sizeForDrawing()->maxDimension();
     }
 
+    DecodeToHDR decodeToHDR() const { return m_decodeToHDR; }
+
 private:
     DecodingMode m_decodingMode;
     std::optional<IntSize> m_sizeForDrawing;
+    DecodeToHDR m_decodeToHDR;
```

Comment:
It's probably better packed up next to `m_decodingMode` ?

Reply:
Moved.